### ISSUE TITLE
IDAPro 7.4 python3.9 crash at capa_explorer loading

### ISSUE THREAD

**Issue by xor-al-ex** [fireeye/capa#726]
### Description
New installation of IDAPro 7.4.191011, originally chose Python2.7, but changed to python 3.9 for this plugin. At startup of IDA, or of manually loading python plugin module "CapaExplorerPlugin", the instance crashes and creates a dump-file. 

### Steps to Reproduce

Install Python3.9 from python.org
a. Run idapyswitch as admin, change .disabled files. 

Install capa explorer as instructed. 

Open any PE-file in IDA to crash.

**Expected behavior:**
Expected to load capa_explorer module.

**Actual behavior:**
IDA crash at python plugin load

### Versions

idapyswitch:
```
PS C:\Program Files\IDA Pro 7.4> .\idapyswitch.exe -v
V: Searching for subkeys of "HLKM\Software\Python"
V: Found "HLKM\Software\Python\PythonCore"
V: Opened "2.7"
V: Not a Python 3.x or no version info, skipping
V: Searching for subkeys of "HKCU\Software\Python"
V: Found "HKCU\Software\Python\PythonCore"
Checking installs from "Python Software Foundation"
V: Opened "3.9"
Checking "Python 3.9 (64-bit)" (3.9)
V: Found: "C:\Users\REM\AppData\Local\Programs\Python\Python39\python3.dll" (version: 3.9.0 ('39'))
V: Found: "C:\Users\REM\AppData\Local\Programs\Python\Python39\python39.dll" (version: 3.9.0 ('39'))
Found: "C:\Users\REM\AppData\Local\Programs\Python\Python39\" (version: 3.9.0 ('39'))
V: Previously-used DLL: "C:\Users\REM\AppData\Local\Programs\Python\Python39\python39.dll"
V: Found: "C:\Users\REM\AppData\Local\Programs\Python\Python39\python3.dll" (version: 3.9.0 ('39'))
V: Found: "C:\Users\REM\AppData\Local\Programs\Python\Python39\python39.dll" (version: 3.9.0 ('39'))
IDA previously used: "C:\Users\REM\AppData\Local\Programs\Python\Python39\python39.dll" (guessed version: 3.9.0 ('39')). Making this the preferred version.
    V:     Version: 3.9.0 ('39'); paths: C:\Users\REM\AppData\Local\Programs\Python\Python39\python3.dll, C:\Users\REM\AppData\Local\Programs\Python\Python39\python39.dll (PREFERRED)
    V:     Version: 3.9.0 ('39'); paths: C:\Users\REM\AppData\Local\Programs\Python\Python39\python3.dll, C:\Users\REM\AppData\Local\Programs\Python\Python39\python39.dll
The following Python installations were found:
    #0: 3.9.0 ('39') (C:\Users\REM\AppData\Local\Programs\Python\Python39\python3.dll)
    #1: 3.9.0 ('39') (C:\Users\REM\AppData\Local\Programs\Python\Python39\python3.dll)
Please pick a number between 0 and 1 (default: 0)
0
Applying version 3.9.0 ('39')
V: Handling sip.pyd
V: Reading import table at 96136 (0x17788)
V: Import table entry #0; dll name: "python39.dll"
V: Found python3Y.dll: "python39.dll" at offset 100846 (0x189ee)
V: File "C:\Program Files\IDA Pro 7.4\python\3\PyQt5\sip.pyd" successfully patched (with "python39.dll")
```
IDAPro version: 7.4.191011

Python version: 3.9.6

Windows version: 10.0.16299 Build 16299 (Remnux Workstation) 

### Additional Information
I have not done any other changes than installing IDAPro, switching py version and followed the install steps of capa-explorer.

**Comment by williballenthin** [comment]
Do you receive any crash details? e.g. from the IDA output window?

**Comment by xor-al-ex** [comment]
Sadly, no. I have attached an image of what happens if I import the capa explorer plugin. The image is after I've entered the command in the python console. The popup is the only output I get before IDA shuts down. 
![capa_explorer_dump](https://user-images.githubusercontent.com/83962312/129524234-1c1e43a2-4d15-453b-9a44-7ac5bb5cfc89.png)

**Comment by mike-hunhoff** [comment]
TLDR; I believe this crash is occurring because, from my understanding, Python 3.9.x is not fully supported until IDA 7.5 SP3 as noted by hex-rays [here](https://hex-rays.com/blog/python-3-9-support-for-ida-7-5/). To use capa explorer with IDA 7.4 and IDA 7.5 < SP3 you will only be able to use up to the latest Python 3.8.x release. To use Python 3.8.x you will also need to install the sip module patch provided by hex-rays [here](https://hex-rays.com/blog/python-3-9-support-for-ida-7-5/).

#### Details

capa explorer makes extensive use of PyQt and it appears this is causing a crash to occur in situations where idapython does not support the Python version being used as described by hex-rays [here](https://hex-rays.com/blog/python-3-9-support-for-ida-7-5/).

I did some testing locally and capa explorer works in IDA 7.4 using Python 3.8.10:
```
---------------------------------------------------------------------------------------------
Python 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)] 
IDAPython 64-bit v7.4.0 final (serial 0) (c) The IDAPython Team <idapython@googlegroups.com>
---------------------------------------------------------------------------------------------
INFO:capa.ida.plugin.form:Analysis completed.
Python>idaapi.get_kernel_version()
'7.4'
```

We will update our plugin documentation with more explicit Python version support for the various IDA versions to avoid further confusion in the future.

**Comment by xor-al-ex** [comment]
Hello, thank you for your time and help. I was finally able to test solutions, and I had to go down to Python 3.6.8 for it to work, but it works!
